.github/workflows/simulation-1.1.yml

```diff
@@ -316,6 +323,8 @@ jobs:
 
   external-commissioner:
     runs-on: ubuntu-20.04
+    env:
+      THREAD_VERSION: 1.1
```

Comment:
But the CI tests on `ot-commissioner` master will be broken.

Reply:
Do you have a suggested path forward?

Reply:
I think I misunderstood this change. I thought there are some external commissioner failures that force you setting the `THREAD_VERSION` to 1.1 for this test case, but seems we are just using 1.1 for all test cases in this workflow and the workflow itself is also renamed to `simulation-1.1.yml`.

Actually, the external-commissioner test script doesn't respect the `THREAD_VERSION` env variable and it actually tests with latest openthread master commit and the CI on `ot-commissioner` master  will conitnue working.